L1Trigger/TrackFindingTracklet/src/VMRouterCM.cc

```diff
@@ -24,10 +31,30 @@ VMRouterCM::VMRouterCM(string name, Settings const& settings, Globals* global)
   overlapbits_ = 7;
   nextrabits_ = overlapbits_ - (settings_.nbitsallstubs(layerdisk_) + settings_.nbitsvmme(layerdisk_));
 
-  meTable_.initVMRTable(layerdisk_, TrackletLUT::VMRTableType::me, region);  //used for ME and outer TE barrel
+  meTable_.initVMRTable(layerdisk_, TrackletLUT::VMRTableType::me, region);            //used for ME and outer TE barrel
+  meTableOld_.initVMRTable(layerdisk_, TrackletLUT::VMRTableType::me, region, false);  //used for ME and outer TE barrel
```

Comment:
Needs a comment to explain why we now have "old" tables in addition to the original ones.

Reply:
Done.